ts/screens/messages/MessageDetailScreen.tsx

```diff
@@ -308,4 +309,8 @@ const mapStateToProps = (
   };
 };
 
-export default connect(mapStateToProps)(MessageDetailScreen);
+export default withLoadingSpinner(
```

Comment:
I don't think it makes sense to add a loading spinner to the message detail screen considering that the message is loaded and displayed, the loading spinner should be displayed on the screen that is waiting for remote data to be available - in this case the payment summary screen. What should happen here is that I click on "pay" and the app navigates to the payment summary and _there_ I see the loading spinner (the next step would be to render the screen with [loading placeholders](https://github.com/mfrachet/rn-placeholder))